source/clear-linux/get-started/virtual-machine-install/vmware-esxi.rst

```diff
@@ -1,80 +1,454 @@
 .. _vmware-esxi:
 
-Use VMware\* ESXi guest
-#######################
+Run Clear Linux as a VMware\* ESXi guest OS
+###########################################
 
+This section explains how to run |CLOSIA| in a virtualized environment using 
+VMware ESXi 6.5 Update 1. 
```

Comment:
> +Run Clear Linux as a VMware\* ESXi guest OS
> +###########################################
  
 >+This section explains how to run |CLOSIA| in a virtualized environment using 
 >+VMware ESXi 6.5 Update 1.

This is great